src/icons/test.ts

```diff
@@ -35,9 +35,9 @@ const testIconDirectory = (directory: string) => {
       path.resolve(__dirname, `${directory}/${SVG_DIRECTORY_NAME}`),
     );
 
-    const associatedSvgIconFilenames = Array.from(new Set(svgFilenames))
-      .map(transformIconFileNames)
-      .sort();
+    const associatedSvgIconFilenames = Array.from(
+      new Set(svgFilenames.map(transformIconFileNames).sort()),
```

Comment:
The uniqueness application (`Array.from(new Set([]))`) should've been happening after the transform, but the test didn't value when initially added because there wasn't a difference yet.